apps/digital-inclusion/models/save.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const mysql = require('mysql');
+const config = require('../../../config');
+
+const pool = mysql.createPool({
```

Comment:
I'm not familiar with best practice for this library but I would suspect that this should be farmed out to a `mysql.js` or `db.js` that exports the pool.

That way you could re-use the pool across models / tables.

Reply:
what you suspect is completely logically.  However, the examples they provide in their docs are a bit more strange.  It seems, I think and I'm not sure.  You create a Pool, then you get a pool by doing getConnection.  If a pool has not been created, mysql creates one.  Otherwise it just reuses one if it is already there.

See documentation here https://github.com/mysqljs/mysql#pooling-connections

So I think what I have is fine but I'm happy to change it if it isn't correct

Reply:
I think what they are demonstrating is that you can share a single connection for multiple consecutive queries for speed. If you are only making one query at a time then you could probably follow the "Use pool directly" style.

You should still be able to move the pool into its own file. (Which would export the pool.) `save.js` could then just require in the pool.

Reply:
I tried to do what you suggested but for some reason I get `cannot read property 'getconnection' of undefined`.  For now, I'll just merge.  I'll investigate this in the future.

Thanks again